config.yml

```diff
@@ -109,12 +109,30 @@ trees:
     quiet_period: 14400
     job_weight: 4
     es_index: 'dxr_hot_{format}_{tree}_{unique}'
+    enabled_plugins: 'pygmentize rust buglink omniglot'
     ignore_patterns: '/compiletest/ /doc/ /etc/ /gyp/ .hg .git CVS .svn .bzr .deps .libs .DS_Store .nfs* *~ ._* *.pyc'
     build_command: 'cd $source_folder && ./configure --disable-libcpp --enable-ccache --enable-clang && make clean && make -j4'
     plugins:
       - plugin: 'buglink'
         name: 'github'
-        url: 'https://github.com/mozilla/rust/issues/%s'
+        url: 'https://github.com/rust-lang/rust/issues/%s'
+        regex: '#([0-9]+)'
+
+  - name: 'rustfmt'
+    proj_dir: 'rustfmt'
+    repos:
+      - url: 'https://github.com/rust-lang-nursery/rustfmt.git'
+    schedule: 'H H/6 * * *'
+    quiet_period: 14400
+    job_weight: 4
```

Comment:
I'm not sure what these mean - it would be totally fine to lower the priority on Rustfmt and to build it less often (it's not a very big project). I just copied the Rust values to start with.